src/Egulias/EmailValidator/EmailValidator.php

```diff
@@ -74,6 +112,26 @@ public function __construct()
         $this->parser = new EmailParser(new EmailLexer());
     }
 
+    /**
+     * Validates an email address against the following standards:
+     *
+     * RFC-5322: Internet Message Format
+     * RFC-5321: Simple Mail Transfer Protocol
+     * RFC-6530: Overview and Framework for Internationalized Email
+     * RFC-1123 section 2.1: Requirements for Internet Hosts -- Application and Support
+     * RFC-4291 section 2.2: IP Version 6 Addressing Architecture
+     *
```

Comment:
Please add

- [6530](http://tools.ietf.org/html/rfc6530#section-4.6)
- [6531](http://tools.ietf.org/html/rfc6531)
- [6532](http://tools.ietf.org/html/rfc6532) 

which are for **SMTPUTF8** international emails.